MasterUI.cxx:2377:3: error: 'fl_open_display' was not declared ...

Other files are using x.H, but that header is deprecated and the
correct one is platform.H. fl_display that it alse needs platform.H
for is X-specific, so this likely cannot work without fltk[X] at
the moment.

(next revision of the ebuild removes fltk, so it won't matter much)
--- a/src/UI/MasterUI.fl
+++ b/src/UI/MasterUI.fl
@@ -76,6 +76,9 @@
 decl {\#include "common.H"} {public local
 } 
 
+decl {\#include "FL/platform.H"} {public local
+} 
+
 decl {\#if USE_NSM
 \#include "NSM.H"
 extern NSM_Client *nsm;
